certbot-ci/certbot_integration_tests/utils/misc.py

```diff
@@ -205,7 +205,7 @@ def manual_http_hooks(http_server_root, http_port):
 import shutil
 well_known = os.path.join('{0}', '.well-known')
 shutil.rmtree(well_known)
-'''.format(http_server_root))
+'''.format(http_server_root.replace('\\', '\\\\')))
```

Comment:
Exactly. As you would write a Windows path in a python script, escaping the backslashes, you want the same result in a generated python script. But since this generation also occurs in python, we need a escape-ception.